package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Immutable",
   "bugs": "https://github.com/immutable/ts-immutable-sdk/issues",
   "dependencies": {
+    "localforage": "^1.10.0",
```

Comment:
This dependency should be in `/sdk/package.json` & `/packages/passport/sdk/package.json`